Cython/Compiler/Parsing.py

```diff
@@ -2818,17 +2788,6 @@ def looking_at_dotted_name(s):
     else:
         return 0
 
-def looking_at_call(s):
-    "See if we're looking at a.b.c("
-    # Don't mess up the original position, so save and restore it.
-    # Unfortunately there's no good way to handle this, as a subsequent call
-    # to next() will not advance the position until it reads a new token.
-    position = s.start_line, s.start_col
-    result = looking_at_expr(s) == u'('
-    if not result:
-        s.start_line, s.start_col = position
-    return result
```

Comment:
This is genuinely unused so is probably worth removing whatever we think of the rest of the changes